datasette_render_markdown/__init__.py

```diff
@@ -17,6 +17,8 @@ def render_cell(value, column, table, database, datasette):
         )
         or {}
     )
+    extensions = config.get("extensions") or []
+    extra_tags = config.get("extra_tags") or []
```

Comment:
`extra_tags` mechanism is necessary because extensions such as `tables` use HTML tags from outside the initial Bleach whitelist. I need to document this mechanism.